src/style.css

```
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  margin-right: 4px;
+}
```

```suggestion
}

.stars__star:not(:last-child) {
  margin-right: 4px
}
```
do not set margin-right for last star in row